bitbot/links_test.go

```diff
@@ -25,7 +25,6 @@ func TestGetHTMLTitle(t *testing.T) {
 		t.Log("unexpected title")
```

Comment:
"unexpected title" is a misleading error message for that test. We're checking to see if the title is longer than we expect from the truncated title the function should be returning.

The function should return _at most_ a slice with a length of 353 (350 for text + 3 for the `...`). `"Title too long. Expected 353, got len(title)"` or something like that would be more helpful